### PR TITLE
[doc][stream-load-manual] Add usage details for importing JSON files to the stream-load-manual documentation

### PR DESCRIPTION
**Pull request by bingquanzhao**
Add usage details for importing JSON files to the stream-load-manual documentation.
Reduce the user’s understanding cost when using stream load to import JSON files.

### CODE REVIEW

**Review by DongLiang-0 — APPROVED** [review]
LGTM

**Review by luzhijing — APPROVED** [review]
Please also fix it in 3.0-branch

File: docs/data-operate/import/stream-load-manual.md
Comment (KassieZ) [inline docs/data-operate/import/stream-load-manual.md:1]: Its recommended to use admonitions (:::info 备注::: ) for note and info

**Review by zy-kkk — APPROVED** [review]

**Review by KassieZ — COMMENTED** [review]
please also update v3.0 of CN docs and dev version of EN docs

**Review by KassieZ — APPROVED** [review]

**Pull request merged**